aicodebot/cli.py

```diff
@@ -23,8 +24,15 @@ def cli(ctx, debug_output):
             configure.callback(openai_api_key=os.getenv("OPENAI_API_KEY"), verbose=0)
             sys.exit(0)
     else:
-        os.environ["OPENAI_API_KEY"] = existing_config["openai_api_key"]
-
+        match LanguageModelManager.CURRENT_PROVIDER:
```

Comment:
We missed the mark here. There was no need to introduce `CURRENT_PROVIDER`. Instead, we should pull the provider from the config file, which is already set to work for this. As an example, here is my config file from when I was testing with HuggingFace

```
openai_api_key: sk-****
openrouter_api_key: sk-or-****
language_model_provider: HuggingFace Hub
language_model: stabilityai/stablecode-completion-alpha-3b
personality: Her
version: 1.1
```

So you should read the `provider` from `existing_config.get('language_model_provider', self.DEFAULT_PROVIDER)` and then the same for model. 

To test, edit your config file and add/edit this line:
language_model_provider: Ollama

If you remove this line, it will default to DEFAULT_PROVIDER (OpenAI)

Then you can switch back and forth easily just by editing the config file